1-Draft/RFCxxxx-PSModulePath.md

```diff
@@ -0,0 +1,159 @@
+---
+RFC:
+Author: Steve Lee
+Status: Draft
+SupercededBy: n/a
+Version: 1.0
+Area: Engine
+Comments Due: 11/30
+Plan to implement: Yes
+---
+
+# PSModulePath When Starting PowerShell within PowerShell
+
+When starting PowerShell from a different version of PowerShell, the PSModulePath
+should reflect the module search path and only include segments appropriate for
+the version of PowerShell being started.
+
+## Motivation
+
+    As a PowerShell User,
+    I can start a different version of PowerShell within PowerShell,
+    so that I can do things required by that specific version of PowerShell.
+
+## User Experience
+
+### Starting Windows PowerShell from PowerShell 7
+
+    ```powershell
+    PS7> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\PowerShell\Modules
+    C:\Program Files\PowerShell\Modules
+    c:\program files\powershell\7-preview\Modules
+    C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
+
+    PS7> powershell -noprofile
+    WinPS> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\WindowsPowerShell\Modules
+    C:\Program Files\WindowsPowerShell\Modules
+    C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
+    C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell
+    ```
+
+### Starting Windows PowerShell from PowerShell 7 with additions by user
+
+    ```powershell
+    PS7> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\PowerShell\Modules
+    C:\Program Files\PowerShell\Modules
+    c:\program files\powershell\7-preview\Modules
+    C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
+    C:\MyModules
+
+    PS7> powershell -noprofile
+    WinPS> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\WindowsPowerShell\Modules
+    C:\Program Files\WindowsPowerShell\Modules
+    C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
+    C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell
+    C:\MyModules
+    ```
+
+### Starting Windows PowerShell from PowerShell 7 with deletions by user
+
+    ```powershell
+    PS7> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\PowerShell\Modules
+    C:\Program Files\PowerShell\Modules
+    c:\program files\powershell\7-preview\Modules
+
+    PS7> powershell -noprofile
+    WinPS> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\WindowsPowerShell\Modules
+    C:\Program Files\WindowsPowerShell\Modules
+    C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
+    C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell
+    ```
+
+### Starting Windows PowerShell from PowerShell 7 with additions and deletions by user
+
+    ```powershell
+    PS7> $env:PSModulePath.split(';')
+    C:\Users\slee\Documents\PowerShell\Modules
+    C:\Program Files\PowerShell\Modules
+    c:\program files\powershell\7-preview\Modules
+    C:\MyModules
+
+    PS7> powershell -noprofile
```

Comment:
please review this - update this to reflect in windows powershell only "MyModules" will show up.